docs_src/design/adr/devops/0010-Release-Artifacts.md

```diff
@@ -2,11 +2,13 @@
 
 ## Status
 
-In Review 06/10/2020
+Original Approved by TSC 07/2020
+
+Updates In Review for Hanoi
```

Comment:
Same comment as for the similar line in ADR0007...

Reply:
I removed any date changes from my PR to avoid any circular references, I'm not sure how we handle this scenario. 

I can add a line back to reference a TSC approval date after the fact if that's the norm.